.gitignore

```diff
@@ -129,3 +129,4 @@ dmypy.json
 .pyre/
 .pytype/
 .idea/
+.vscode/
```

Comment:
We should keep the .vscode settings in the repo, so that we can always load the correct settings for our python project.

Reply:
Yes, we should keep this `.vscode` settings in this repo. I will modify it in next commit.